docs/explanation/settings-keys-values.md

```diff
@@ -0,0 +1,18 @@
+(explanation-settings-keys-values)=
+# Settings keys and values
+
+> See also: [Settings](/reference/settings/index)
+
+Multipass settings are organised in a tree structure, where each individual setting is identified by a unique **key** and takes on a single **value** at any given time.
+
+## Settings keys
+
+A **settings key** is a string in the form of a dot-separated path through the settings tree (such as `client.primary-name`). It specifies a path along the settings tree, from the root to a leaf. Thus, individual settings correspond to the leaves of the settings tree.
+
+Conceptually, branches of the tree can be singled out with wildcards, to refer to multiple settings at once. For instance, `local.<instance-name>.*` designates the settings that affect a specific instance. Wildcards can also be used to select separate branches. For example `local.*.cpus` refers to the number of CPUs of Multipass instances.
+
+## Settings values
+
+A **settings value** is a string whose syntax (possible values/representations) and semantics (their interpretation) is determined by the setting in question.
```

Comment:
In `possible values/representations`, I'd rewrite the slash because of the inherent ambiguity of slashes:

```
possible values or their representations
```